docs/en/docs/kpanda/user-guide/workloads/create-deployment.md

```diff
@@ -155,11 +178,10 @@ Advanced setting includes four parts: load network setting, upgrade strategy, sc
 
 === "Scheduling Policies"
 
-     - Tolerance time: When the node where the load instance is located is unavailable, the time for rescheduling the load instance to other available nodes, the default is 300 seconds.
-     - Node affinity: According to the label on the node, constrain which nodes the Pod can be scheduled on.
+     - Toleration time: When the node where the load instance is located is unavailable, the time for rescheduling the load instance to other available nodes, the default is 300 seconds.
+     - Node Affinity: According to the label on the node, constrain which nodes the Pod can be scheduled on.
      - Workload Affinity: Constrains which nodes a Pod can be scheduled to based on the labels of the Pods already running on the node.
-     - Workload anti-affinity: Constrains which nodes a Pod cannot be scheduled to based on the labels of Pods already running on the node.
-     - Topology domain: namely topologyKey, used to specify a group of nodes that can be scheduled. For example, __kubernetes.io/os__ indicates that as long as the node of an operating system meets the conditions of labelSelector, it can be scheduled to the node.
+     - Workload Anti-affinity: Constrains which nodes a Pod cannot be scheduled to based on the labels of Pods already running on the node.
     
      > For details, refer to [Scheduling Policy](pod-config/scheduling-policy.md).
```

Comment:
The image below shall be indented with only 4 spaces instead of current 8 spaces